test/integration-tests/test-menu-popup.c

```diff
@@ -6,7 +6,7 @@ static const char *options[] = {"Foo", "Bar", "Baz", NULL};
 
 static void callback_0()
 {
-    EXPECT_MESSAGE(zwlr_layer_shell_v1 .get_layer_surface);
+    EXPECT_MESSAGE(zwlr_layer_shell_v1 .get_layer_surface wl_output#);
```

Comment:
Adding this `wl_output` might be an erroneous change, that was causing your problems with this test